src/patternfly/components/Login/login.scss

```diff
@@ -117,7 +116,6 @@
 }
 
 .pf-c-login__container {
-  width: 100%;
```

Comment:
This change resulted in a regression where the container can be smaller than the viewport at mobile.

![oauth-openshift apps rhamilto devcluster openshift com_oauth_authorize_client_id=console redirect_uri=https%3A%2F%2Fconsole-openshift-console apps rhamilto devcluster openshift com%2Fauth%2Fcallback response_type=code scope=user%3Afull  (1)](https://user-images.githubusercontent.com/895728/78154827-b990f000-740a-11ea-903f-919ec6296318.png)

![www patternfly org_v4_documentation_react_components_loginpage_basic(iPhone 6_7_8)](https://user-images.githubusercontent.com/895728/78155089-05439980-740b-11ea-8e0c-9a57773ed72b.png)


I'm pretty sure we're going to want this back.

Reply:
Thanks for catching that Robb, I'll try and look at https://github.com/patternfly/patternfly/issues/2913 later today.

Reply:
I've put up a PR and added a comment to the code so hopefully that shouldn't be removed again. https://github.com/patternfly/patternfly/pull/2917